addons/mass_mailing/controllers/main.py

```diff
@@ -6,7 +6,8 @@
 
 from odoo import _, exceptions, http, tools
 from odoo.http import request
-from odoo.tools import consteq
+from odoo.tools import consteq, authenticate_message
```

Comment:
Actually tools is already imported, you can use tools.authenticate_message (and tools.consteq if used in this file).